app/boards/shields/zodiark/Kconfig.defconfig

```
@@ -0,0 +1,62 @@
+# Copyright (c) 2021 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_ZODIARK_LEFT
+
+config ZMK_KEYBOARD_NAME
+	default "Zodiark Left"
```

We just merged #658 which is moving away from advertising the name of the right side to avoid confusion, so please remove the left here, e.g.:

```suggestion
	default "Zodiark"
```